daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md

```diff
@@ -0,0 +1,15 @@
+---
+type: docs
+title: "How to: Use the Dapr Kubernetes Operator"
+linkTitle: "Dapr Kubernetes Operator"
+weight: 7000
+description: "Use the Dapr Kubernetes Operator to manage the Dapr lifecycle"
+---
+
+The Dapr Kubernetes Operator manages the lifecycle for Dapr and its components. Use the operator to automate the tasks required when oeprating Dapr in Kubernetes mode. 
+
+{{% alert title="Note" color="primary" %}}
+The Dapr Kubernetes Operator is currently a Dapr sandbox project.
```

Comment:
You can remove this comment. It should be in dapr org as per https://github.com/dapr/community/issues/425